src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1568,6 +1575,28 @@ public override BoundNode VisitCall(BoundCall node)
                 ReplayReadsAndWrites(localFunc, node.Syntax, writes: true);
             }
 
+            string key = ExtraAnnotations.MakeMethodKey(method);
```

Comment:
>string key = ExtraAnnotations.MakeMethodKey(method); [](start = 12, length = 52)

I think all annotations should come from symbols, this component shouldn't even know that such thing as ExtraAnnotations exist.